corehq/apps/users/forms.py

```diff
@@ -1436,3 +1477,25 @@ def clean_domains(self):
             domains = EnterprisePermissions.get_domains(self.domain)
             domains += [self.domain]
         return sorted(domains)
+
+    def clean_user_active_status(self):
+        user_active_status = self.cleaned_data['user_active_status']
+
+        if user_active_status == self.ACTIVE:
+            return True
+        if user_active_status == self.INACTIVE:
+            return False
+        return None
+
+    def get_cleaned_data(self):
```

Comment:
`Form.clean()` already exists to do what you're doing here. See [Django docs](https://docs.djangoproject.com/en/2.2/ref/forms/validation/):

> The form subclass’s clean() method can perform validation that requires access to multiple form fields.

```suggestion
    def clean(self):
```

The benefits of using `clean()`:
* It uses the Django idiom
* It gets called automatically
* `form.cleaned_data` will already have "web_user_assigned_location_ids" set when you need it in `corehq/apps/users/views/mobile/users.py`